### ISSUE TITLE
Add dictinary based tooltips

### ISSUE THREAD

**Issue by ThomDietrich** [openhab/openhab-docs#538]
We should implement some javascript trickery to replace words by html code to include CSS tooltips (one possible solution).

See: https://www.w3schools.com/css/css_tooltip.asp

Possible use cases: Giving more details about e.g. `$OPENHAB_CONF`: "The location of all configuration files. See the platform specific installation articles to learn more."

Wdyt?

**Comment by Confectrician** [comment]
I agree, this could be useful in some/many cases👍 

But i think there are more urgent things to do, we shouldn't forget while opening new tasks.
- Versioned docs
- Mobile usability
- tbc

I know its a bit off-topic, but we shouldn't forget about these. :)

**Comment by ThomDietrich** [comment]
Good point! All of this was part of long discussions over the weekend (e.g. with @kubawolanin) and I intend to post a requirements list for the webpage rework in the next couple of days.

Besides that: It is okay to write down an unimportant idea even if no one intends to work on it in the near future.

**Comment by Confectrician** [comment]
> Besides that: It is okay to write down an unimportant idea even if no one intends to work on it in the near future.

Just to be sure: 
I didn't want to connote, that this is unimportant. 😃 
And of course I am a friend of writing down ideas.
That's the only way to keep them in mind long term, if you don't work on them directly.

**Comment by bgilmer77** [comment]
It would be good to add this.  The example you give is a great one.  If I had not been working with OH for a while, it would take some digging to figure out what `$OPENHAB_CONF` meant and what it was for my system.  A simple tool tip like this might keep someone from walking away from OH all together.  Remember - it has to be easy to adopt OH or people go to commercial solutions with better support.  IMHO, it is still a very tall hill to climb to get running with OH.  Simple things like the tool tips could help.

**Comment by Confectrician** [comment]
Just throwing this in for documentation:

https://github.com/adamdehaven/vuepress-plugin-custom-tooltip

Probably we may maintain a dictionary in a mapping file and place the tooltips in a ruby script during build time.
